### DISCUSSION TITLE
Game Save Synching between Pocket and MiSTer

### DISCUSSION THREAD

**Discussion by RetroDriven** [RetroDriven/Pocket_Updater#7] [Ideas]
Add an option to Sync between your Pocket and MiSTer Game Save file.

- Saves to/from UNC/Local

**Comment by kurohouou** [comment]
Really looking forward to this option to sync the saves on my Mister SMB network share and my Pocket!  One suggestion I might make is give an option to always notify before synchronizing, or have a popup that notifies you what will actually sync/overwrite an existing save either locally or on the network share.  Just incase you don't want to proceed.

**Comment by RetroDriven** [comment]
I made a custom Mister Script that would only backup saves to a SMB/NAS if the file was newer. I think I could do something similar with this as well as have an option to backup saves instead of overwrite them as a failsafe.

So instead of just overwriting Saves in either direction you'd have backups like **Game-date-sav.bak** or something along those lines. Since the Save files are small that's probably worth doing vs potentially losing Saves files completely.
